easybuild/easyconfigs/b/bzip2/bzip2-1.0.6-GCC-5.2.0.eb

```diff
@@ -0,0 +1,19 @@
+name = 'bzip2'
+version = '1.0.6'
+
+homepage = 'http://www.bzip.org/'
+description = """bzip2 is a freely available, patent free, high-quality data compressor. It typically 
+compresses files to within 10% to 15% of the best available techniques (the PPM family of statistical 
+compressors), whilst being around twice as fast at compression and six times faster at decompression."""
+
+toolchain = {'name': 'GCC', 'version': '5.2.0'}
+toolchainopts = {'pic': True}
+
+sources = [SOURCE_TAR_GZ]
+source_urls = ['http://www.bzip.org/%(version)s/']
+
+import os
+os.environ['CC'] = 'gcc'
+os.environ['CFLAGS'] = '-O3 -fPIC'
```

Comment:
This should only be present in the `dummy` bzip2 easyconfig. 

But this easyconfig can go if you switch to the `GNU` toolchain. It already has a bzip2 easyconfig.